spring/boot2-autoconfigure/src/main/java/com/linecorp/armeria/internal/spring/ArmeriaConfigurationUtil.java

```diff
@@ -102,7 +109,9 @@ public static void configureServerWithArmeriaSettings(ServerBuilder server, Arme
 
         final String healthCheckPath = settings.getHealthCheckPath();
         if (!Strings.isNullOrEmpty(healthCheckPath)) {
```

Comment:
How about leaving a warn message if `healthCheckPath` is empty and `healthCheckServiceConfigurators` is not empty such as:
```java
logger.warn("{}s exist but they are disabled by the empty 'health-check-path' property. configurations: {}", 
    HealthCheckServiceConfigurator.class.getSimpleName(), healthCheckServiceConfigurators);
```

Reply:
Nice idea 👍